.werft/jobs/build/prepare.ts

```diff
@@ -74,19 +72,9 @@ function configureStaticClustersAccess() {
     }
 }
 
-async function issueCertificate(werft: Werft, config: JobConfig): Promise<boolean> {
-    const certName = `harvester-${previewNameFromBranchName(config.repository.branch)}`;
-    const domain = `${config.previewEnvironment.destname}.preview.gitpod-dev.com`;
-
-    werft.log(prepareSlices.ISSUE_CERTIFICATES, prepareSlices.ISSUE_CERTIFICATES);
-    var certReady = await issueMetaCerts(werft, certName, "certs", domain, config.repository.branch, prepareSlices.ISSUE_CERTIFICATES);
-    werft.done(prepareSlices.ISSUE_CERTIFICATES);
-    return certReady
-}
-
 async function decideHarvesterVMCreation(werft: Werft, config: JobConfig) {
     // always try to create - usually it will be no-op, but if tf changed for any reason we would reconcile
-    if (config.withPreview) {
+    if (config.withPreview && (!vmExists({ name: config.previewEnvironment.destname }) || config.cleanSlateDeployment)) {
```

Comment:
Re-adding this check until all preview envs have the cert created with tf, so it doesn't fail on reapply.